attributes/default.rb

```diff
@@ -3,6 +3,5 @@
 node.default[:cyclesafe_chef][:group] = 'cyclesafe'
 node.default[:cyclesafe_chef][:directory] = '/srv/cyclesafe'
 node.default[:cyclesafe_chef][:requirements_file] = 'prod_requirements.txt'
-node.default[:cyclesafe_chef][:repository] = 'https://github.com/zemadi/CycleSafe_deploy.git'
```

Comment:
Removing this makes the value of node[:cyclesafe_chef][:repository] to "nil" which, when passed to the application provider will prevent it from deploying the django infrastructure. This means that we can deploy solely system settings without interrupting the app, and we can then pass custom deployments to the server when we are ready.